yaks/spec/unit/yaks/default_policy/derive_mapper_from_object_spec.rb

```diff
@@ -3,16 +3,79 @@
 
   let(:options) { {} }
 
+  it 'should use const_get with second argument set to false' do
+    expect(Object).to receive(:const_get).with(kind_of(String), false)
```

Comment:
@plexus `kind_of(String)` is not really necessary here. It's just that I could not make `any_args` to work properly with rspec-mocks.